frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -400,20 +414,23 @@ class EditEmail extends Component {
                 )}
               </label>
               <OverlayTrigger
-                trigger="focus"
+                trigger="click"
                 placement="right"
                 overlay={
-                  <Popover id="reasons-for-action-tooltip">
+                  <Popover>
                     <div>
                       <p>{LOCALIZE.emibTest.inboxPage.addEmailResponse.reasonsForActionTooltip}</p>
                     </div>
                   </Popover>
                 }
               >
                 <Button
+                  id="reasons-for-action-tooltip-button"
+                  tabIndex="-1"
                   aria-label={LOCALIZE.ariaLabel.reasonsForActionTooltip}
                   style={styles.tooltipButton}
                   variant="link"
+                  onBlur={this.triggerReasonTooltipClick}
```

Comment:
Exactly the same comments about the `trigger` and the `onBlur` here.

Reply:
changed in ea161a4